recipes/aiocontextvars/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "aiocontextvars" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1e0ff5837c8b01c36a1107acdd0baf7853ebdf6c9fc43e8e311f4be37ac2038a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - pytest-runner
+  run:
+    - python >=3.5
+    - contextvars ==2.3  # [py<37]
+    - contextvars  # [py>=37]
```

Comment:
The tests are failing as `contextvars` is part of the standard library in Python 3.7+ so there is no need to list it as a dependency.
```suggestion
```

Reply:
I'm not familiar with this package but it might make sense to add `skip: True  # [py>=37]`. The databases package in #7777 only [requires it for older python versions](https://github.com/encode/databases/blob/master/setup.py#L51).

Reply:
(Sorry for the storm of messages, typing without thinking)

Using `python >=3.5,<3.7` would allow this to stop using selectors and have `noarch: python`.

Reply:
thanks @chrisburr just tried your last suggestion!

Reply:
That seems to have done it, LGTM 👍

Reply:
🎉